Library/Formula/kettle.rb

```diff
@@ -55,7 +55,6 @@ def plist; <<-EOS.undent
 
   test do
     ENV["_JAVA_OPTIONS"] = "-Duser.home=#{testpath}"
-    system "#{bin}/pdikitchen", "-file=#{libexec}/samples/jobs/Slowly\ Changing\ Dimension/create\ -\ populate\ -\ update\ slowly\ changing\ dimension.kjb", "-level=RowLevel"
```

Comment:
It no longer passes, even though the install of PDI kettle 6 works perfectly. The problem is with that job they ship with the 6.0 package.